packages/use-pagination/README.md

```diff
@@ -0,0 +1,146 @@
+<p align="center">
+
+<img width="250" src="./hooks.svg" alt="Two fishing hooks">
+
+</p>
+
+# @openformation/use-pagination
+
+> Setup a pagination context and exposes helper to control
+
+## Installation
+
+Using npm:
+
+```
+npm install --save @openformation/use-pagination
+```
+
+Using yarn:
+
+```
+yarn add @openformation/use-pagination
+```
+
+Using pnpm:
+
+```
+pnpm add --save @openformation/use-pagination
+```
+
+## API
+
+```typescript
+usePagination = (spec: {
+  initial?: number,
+  min?: number,
+  max?: number,
+  steps?: number,
+}) => {
+  current: number,
+  isCurrent: (checkValue: number) => boolean,
+  setCurrent: (nextValue: number) => void,
+  next: () => void,
+  prev: () => void,
+  hasNext: () => boolean, 
+  hasPrev: () => boolean, 
+}
+```
+
+`initial` is the page value, pagination will start to work at. If value hurts boundaries given by `min` and/or `max`, it internally becomes adjusted to fit closest boundary.
+*Optional, default value is 0.*
+
+`min` & `max` define boundaries for page value. 
+*Optional, default values are "undefined"*
```

Comment:
Wouldn't it be easier to let `min` default to `-Infinity` and `max` to `Infinity`? I'd imagine that to spare some `if`-clauses at least :)

Reply:
Hi @grebaldi, you are right. I have adjusted this in code and Readme. This improves code readability a lot. I also have "extended" the use of the `keepValueInsideRange` for `prev()` and `next()`. This is the thing, that will happen at each state update.